Brokerages/BaseWebsocketsBrokerage.cs

```diff
@@ -71,6 +67,8 @@ public abstract class BaseWebsocketsBrokerage : Brokerage
         private readonly object _lockerConnectionMonitor = new object();
         private volatile bool _connectionLost;
         private const int _connectionTimeout = 30000;
+
+        protected DataQueueHandlerSubscriptionManager _subscriptionManager;
```

Comment:
code style nit - missing xml docs and should be renamed to `SubscriptionManager`

Reply:
done